backend/src/common/runner/index.ts

```diff
@@ -144,10 +151,13 @@ export const queueJob = (job: Job) : Promise<JobResult> => {
     });
 };
 
-const create_images = () => {
-    console.log("creating images");
-    const img = 'python';
-    const img_ver = '3.10.11';
+// Add new images here
+export const images : RunnerImage[] = [

```

Comment:
feels like this could be a json file

Reply:
or like dynamically generated

Reply:
Probably fine as is for now, we don't really do config files for backend